packages/realm-react/package.json

```diff
@@ -10,7 +10,7 @@
     "build": "wireit",
     "start": "npm run build -- --watch",
     "test": "jest --forceExit",
-    "lint": "eslint --ext .tsx . && tsc --noEmit"
```

Comment:
@takameyer please note this change. Since we're running the rollup bundler with `noEmitOnError` I don't see a need to invoke the transpiler (which would fail on CI since `realm` isn't yet bundled) when linting.

Reply:
The strange thing is that I do remember already removing this at some other point in time / PR 🤔

Reply:
Yeah...I also remember that, but not sure where/when that was.